.github/workflows/build.yml

```
@@ -58,8 +58,19 @@ jobs:
         if: always()
         run: |
           mkdir logs
-          cd /tmp/pytest-of-firedrake/pytest-0/
+          cd /tmp/pytest-of-firedrake/pytest-*/
           find . -name "*.log" -exec cp --parents {} /__w/gusto/gusto/logs/ \;
+      - name: Gusto jupyter_notebooks
+        run: |
+          . /home/firedrake/firedrake/bin/activate
+          export GUSTO_PARALLEL_LOG=FILE
+          python -m pytest \
+            -n 4 --dist loadscope \
+            --timeout=1200 \
+            --timeout-method=thread \
+            -o faulthandler_timeout=1260 \
+            --nbval-lax \
```

```suggestion
            --nbval \
```
When you're happy with the state of the notebooks, you can drop the `lax` and the test will check that the output doesn't change. Use `# NBVAL_IGNORE_OUTPUT` for specific cells where this is undesirable, see [these docs](https://nbval.readthedocs.io/en/latest/#Avoid-output-comparison-for-specific-cells).